pkg/commands/deploy.go

```diff
@@ -37,15 +42,53 @@ func NewDeployCommand(commonOpts *CommonOptions) *cobra.Command {
 }
 
 func NewRemoveCommand(commonOpts *CommonOptions) *cobra.Command {
-	//	opts := &deployOptions{}
+	opts := &deployOptions{}
 	remove := &cobra.Command{
 		Use:   "remove",
 		Short: "remove the components and configurations needed for topology-aware-scheduling",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// TODO: remove components here
+			if err := Remove(opts); err != nil {
+				return err
+			}
 			return nil
 		},
 		Args: cobra.NoArgs,
 	}
 	return remove
 }
+
+func Deploy(opts *deployOptions) error {
```

Comment:
we will need to reshape this code a bit when the ongoing refactoring (not even a PR posted) of the `deployer` package lands, but for now looks good enough.

There is no logging/explicit feedback of the operations, but we can fix in a later PR.